README.md

```diff
@@ -2794,6 +2839,53 @@ const fn = ( lorem, sit ) => {};
  */
 const fn = ( a, b ) => {};
 // "jsdoc/check-line-alignment": ["error"|"warn", "always"]
+
+/**
+ * @param {string} lorem Description
+ *   with multiple lines.
+ */
+function quux () {
+}
+// "jsdoc/check-line-alignment": ["error"|"warn", "any",{"wrapIndent":"  "}]
+
+/**
+ * @param {string} lorem Description
+ *   with multiple lines.
+ */
+function quux () {
+}
+// "jsdoc/check-line-alignment": ["error"|"warn", "never",{"wrapIndent":"  "}]
+
+/**
+ * Function description.
+ *
+ * @param {string} lorem Description.
+ * @param {int}    sit   Description multi
+ *                         line with asterisks.
+ */
+const fn = ( lorem, sit ) => {}
+// "jsdoc/check-line-alignment": ["error"|"warn", "always",{"wrapIndent":"  "}]
```

Comment:
Does it make sense indenting like this when using "always"? Maybe users want to indent from the start?

Reply:
If users will go to the trouble of wanting alignment of columns, I figure they will wish to continue this as text wrapping across the columns would be distracting.

Reply:
@brettz9 yes most likely. For me alignment of columns is a bad practice for many reasons (e.g. git history), but each to his own.